engines/sci/sci.cpp

```diff
@@ -436,7 +436,21 @@ Common::Error SciEngine::run() {
 	}
 
 	if (getGameId() == GID_GK2 && ConfMan.getBool("subtitles") && !_resMan->testResource(ResourceId(kResourceTypeSync, 10))) {
-		suggestDownloadGK2SubTitlesPatch();
+		suggestDownloadPatch(_("GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp.\n\n"
```

Comment:
My personal preference would be to spell out "Gabriel Knight 2" instead of "GK2", but I'm not stuck on the idea.